docs/src/pages/docs/comparison.md

```diff
@@ -12,40 +12,47 @@ Feature/Capability Key:
+| Stale While Revalidate                       | ✅                                     | ✅                         | 🛑                                    | ✅                                    |
+| Stale Time Configuration                     | ✅                                     | 🛑                         | 🛑                                    | ✅                                    |
+| Window Focus Refetching                      | ✅                                     | ✅                         | 🛑                                    | 🛑                                    |
+| Network Status Refetching                    | ✅                                     | ✅                         | ✅                                    | 🛑                                    |
+| Cache Dehydration/Rehydration                | ✅                                     | 🛑                         | ✅                                    | ✅                                    |
+| React Suspense (Experimental)                | ✅                                     | ✅                         | 🛑                                    | ✅                                    |
+| Automatic Refetch after Mutation<sup>4</sup> | 🔶                                     | 🔶                         | ✅                                    | ✅                                    |
+| Normalized Caching<sup>5</sup>               | 🛑                                     | 🛑                         | ✅                                    | ✅                                    |
+| GraphQL Fragments                            | 🛑                                     | 🛑                         | ✅                                    | ✅                                    |
+| GraphQL Subscriptions                        | 🛑                                     | 🛑                         | ✅                                    | ✅                                    |
+| Persisted Queries (APQ)                      | 🛑                                     | 🛑                         | ✅                                    | ✅                                    |
+| GraphQL Connections                          | 🛑                                     | 🛑                         | ✅                                    | ✅                                    |
+| Data masking                                 | 🛑                                     | 🛑                         | 🛑                                    | ✅                                    |
+| Generated type safety for input & response   | 🛑                                     | 🛑                         | 🟡                                    | 🟡                                    |
+| Offline support                              | 🛑                                     | 🛑                         | 🛑                                    | 🟡                                    |
```

Comment:
Does it make sense to add a row about moving runtime overhead to compile time? Which is one of the reasons people both like and dislike Relay.

Reply:
Yes that's a good idea but I don't a good for it. Optimized Runtime maybe?

Reply:
Perhaps ‘Compile-time optimisations’ ?

Reply:
I think that's perfect